gap/oper.gi

```diff
@@ -1684,3 +1684,43 @@ function(digraph, list)
 
   return DigraphShortestDistance(digraph, list[1], list[2]);
 end);
+
+#
+
+InstallMethod(DigraphClosure,
+"for a digraph and an integer",
+[IsDigraph, IsPosInt],
+function(digraph, k)
+  local out, adj_mat, degs, n, new_edge, row, i, j;
+
+  if not IsSymmetricDigraph(digraph) or DigraphHasLoops(digraph)
+      or IsMultiDigraph(digraph) then
+    ErrorNoReturn("Digraphs: DigraphClosure: usage,\n",
+      "the digraph must by symmetric, without loops, and no multiple edges,");
+  fi;
+
+  out := [];
+  adj_mat := BooleanAdjacencyMatrix(digraph);
```

Comment:
Not something you need to do anything about, but since we have `AdjacencyMatixMutableCopy` now, I think we should also have `BooleanAdjacencyMatixMutableCopy`.